esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -617,8 +622,9 @@ class CWsWuFileHelper
     CWsWuFileHelper(IPropertyTree *_directories) : directories(_directories) {};
 
     void createWUZAPFile(IEspContext &context, Owned<IConstWorkUnit> &cwu, CWsWuZAPInfoReq &request,
-        StringBuffer &zipFileName, StringBuffer &zipFileNameWithPath);
-    IFileIOStream* createWUZAPFileIOStream(IEspContext &context, Owned<IConstWorkUnit> &cwu, CWsWuZAPInfoReq &request);
+        StringBuffer &zipFileName, StringBuffer &zipFileNameWithPath, unsigned _getThorSlaveLogThreadPoolSize);
+    IFileIOStream* createWUZAPFileIOStream(IEspContext &context, Owned<IConstWorkUnit> &cwu,
```

Comment:
not new: see other comment re. not using Owned<..> &obj parameters.

Reply:
Will switch to a pointer.